src/frontend/Pretty_printing.ml

```diff
@@ -362,6 +364,8 @@ let rec pp_transformed_type ppf (st, trans) =
     | CholeskyCov -> pf ppf "cholesky_factor_cov%a" cov_sizes_fmt ()
     | Correlation -> pf ppf "corr_matrix%a" cov_sizes_fmt ()
     | Covariance -> pf ppf "cov_matrix%a" cov_sizes_fmt ()
+    | StochasticColumn -> pf ppf "stochastic_column_matrix%a" sizes_fmt ()
+    | StochasticRow -> pf ppf "stochastic_row_matrix%a" sizes_fmt ()
```

Comment:
I think these need to be updated -- worth adding these types to a test that gets pretty-printed rather than just code-gen'd